packages/flutter/lib/src/widgets/magnifier.dart

```diff
@@ -651,7 +651,7 @@ class _RenderMagnification extends RenderProxyBox {
           magnificationScale * ((focalPointOffset.dx * -1) - thisCenter.dx) + thisCenter.dx,
           magnificationScale * ((focalPointOffset.dy * -1) - thisCenter.dy) + thisCenter.dy)
       ..scale(magnificationScale);
-    final ImageFilter filter = ImageFilter.matrix(matrix.storage, filterQuality: FilterQuality.high);
```

Comment:
I think this one should actually still be high. The other use case for cubic is really, really high scales which the magnifier will do.

Reply:
Reverted back to high.